portfolio_style.css

```diff
@@ -0,0 +1,155 @@
+@import url('https://fonts.googleapis.com/css?family=Charm|Fredericka+the+Great');
+
+html {
+	font-family: "Fredericka the Great";
+	color: #5f6161;
+	font-size: 10pt;
+	
+}
+
+*, *::before, *::after {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+	
+body {
+	background-color: white;
+}
+
+a {
+	color: white;
+	text-align: center;
+}
+
+a:hover {
+	color: #e00dbd
+}
+
+a:focus{
+	color:#e00dbd;
+}
+
+p {
+	text-align: center;
+}
+
+h1, h2 {
+	color: #b5b0b3;
+	text-align: center;
+}
+
+h1 {
+	font-size: 50pt;
```

Comment:
should use rem here. not sure how pt converts to px but if it was 50px here you would put 5rem instead.

Reply:
I changed all fonts to rem as you suggested. Good one!